RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -30,17 +25,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     using ::lst::Params_T3;
     using ::lst::Params_T5;
     using ::lst::size_superbins;
```

Comment:
I'm just curious - why not simply `using namespace ::lst` ?

Reply:
I'm fixing that in #89